Mathlib/NumberTheory/LSeries/Linearity.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2024 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.NumberTheory.LSeries.Basic
+
+/-!
+# Linearity of the L-series of `f` as a function of `f`
+
+We show that the `LSeries` of `f : ℕ → ℂ` is a linear function of `f` (assuming convergence
+of both L-series when adding two functions).
+-/
+
+/-!
+### Addition
+-/
+
+open LSeries
+
```

Comment:
You can probably set f,g and some of a,b,c,s as implicit variables at the top and save having to create them each time.

Reply:
I don't think this would be an improvement (see [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/'variable'.20-.20style.20question/near/425503092)).